### PR TITLE
dd: Fix path to delete and prefix template with hostname

### PR DESCRIPTION
**Pull request by nbarithel**

### CODE REVIEW

File: yml/roles/alfresco/handlers/main.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@
 - name: Clean dd temp perf files (Repository)
   file:
     state: absent
-    path: "{{ contentstore_directory | default(contentstore_dir) }}/.hc_tmp_dd_perf_file.raw"
+    path: "{{ contentstore_directory | default(contentstore_dir.stdout_lines[0] | regex_replace('\\${dir.root}',dir_root.stdout_lines[0])) }}/.hc_tmp_dd_perf_file_{{ inventory_hostname }}.raw"
```
Comment (alxgomz) [inline yml/roles/alfresco/handlers/main.yml]: Good catch! But is there any reason for using double backslash? The rest of the playbook seems to use a single escaped backslash

**Comment by alxgomz** [comment]
Taking a look at your PR I wonder what's the point of using unique filenames for the dd output test file...
I thought you meant to make the md resulting file recognizable rather than the raw file.
Am I missing something?

File: yml/roles/alfresco/handlers/main.yml (hunk 1 above)
Comment (nbarithel) [reply yml/roles/alfresco/handlers/main.yml]: It is required because it is inside a double quoted string

**Review by alxgomz — APPROVED** [review]
LGTM

**Pull request merged**